db/migrate/20160626214555_create_ratings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRatings < ActiveRecord::Migration
+  def change
+    create_table :ratings do |t|
+      t.decimal :rating
+      t.references :user, index: true, foreign_key: true
+      t.references :resource, index: true, foreign_key: true
+      t.references :service, index: true, foreign_key: true
```

Comment:
I'm curious if what we've got planned for the UX for ratings and reviews: what's that interaction going to look like, and are we building both in the front-end immediately?